css/styles.css

```diff
@@ -386,3 +395,139 @@ textarea {
   background-color: #2230d2;
   color: var(--white);
 }
+
+.large-card {
+  display: none;
+}
+
+@media screen and (min-width: 768px) {
+  .headline-section {
+    height: 100vh;
+    background: var(--white) url(/images/background.svg);
+    object-fit: cover;
+    background-size: 100%;
+    background-repeat: no-repeat;
+    border-bottom-left-radius: 75px;
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+    padding-left: 10rem;
+  }
+
+  .navigation-bar {
+    list-style: none;
+    display: flex;
+    gap: 2rem;
+    justify-content: space-between;
+    margin-right: 8rem;
+  }
+
+  .navigation-bar a {
+    text-decoration: none;
+    color: #344563;
+  }
+
+  .fa-bars {
+    display: none;
+  }
+
+  .card {
+    display: none;
+  }
+
+  .large-card {
+    display: grid;
+    grid-template-columns: repeat(2, 1fr);
+    background: var(--white);
+    border-radius: 20px;
+    margin-bottom: 10rem;
+    padding-bottom: 2rem;
+    width: 80vw;
+  }
+
+  .card-image {
+    padding: 2rem;
+  }
+
+  .categories li {
+    color: #6070ff;
+  }
+
+  .about-page {
+    display: grid;
+    grid-template-columns: repeat(2, 1fr);
+    max-width: 100%;
+  }
+
+  .card-body h2 {
+    font-size: 4rem;
+  }
+
+  .card-body p {
+    font-size: 1.6rem;
+  }
+
+  .languages {
+    display: flex;
+    flex-direction: row;
+  }
+
+  .languages li {
+    width: 122px;
+    height: 120px;
+    background-color: #f7f7f9;
+    margin-top: 0.8rem;
+    gap: 1.5rem;
+    text-align: center;
+    justify-content: center;
+    align-items: center;
+    flex-wrap: wrap;
+  }
+
+  .contact-section {
+    height: 100vh;
+    background: #6070ff url(/images/contactform.svg);
+    object-fit: cover;
+    background-size: 100%;
+    background-repeat: no-repeat;
+    border-top-left-radius: 75px;
+    color: var(--white);
+    width: 100%;
+    margin-top: 0;
+  }
+
```

Comment:
Excellent job fixing the background as stated by the previous reviewer 💯 

[OPTIONAL] The description text of the contact section is not centered properly like in the Figma template. The width is way larger, it must slightly be bigger than the input fields, but not too big.

Your design:
![image](https://user-images.githubusercontent.com/38088818/165929856-4e9c3a7a-9565-4563-8e19-2e18db012cf5.png)

Figma:
![image](https://user-images.githubusercontent.com/38088818/165929512-a9e6afea-b5de-4511-9367-d06235819873.png)